src/api/encoder.py

```diff
@@ -0,0 +1,37 @@
+"""Encoding Utility Logic."""
+
+# Third-Party Libraries
+from connexion.apps.flask_app import FlaskJSONEncoder  # type: ignore
+import six  # type: ignore
+
+# cisagov Libraries
+from api.models.base_model_ import Model
+
+
+class JSONEncoder(FlaskJSONEncoder):
+    """Encoding Class."""
+
+    include_nulls = False
+    encoding_type = "UTF-8"
+
+    def default(self, encode_target):
+        """Use default encooding logic."""
+        if isinstance(encode_target, Model):
+            dikt = {}
+            for attr, _ in six.iteritems(encode_target.swagger_types):
+                value = getattr(encode_target, attr)
+                if value is None and not self.include_nulls:
+                    continue
+                attr = encode_target.attribute_map[attr]
+                dikt[attr] = value
+            return dikt
+        return FlaskJSONEncoder.default(self, encode_target)
+
+    def set_decoding(self, encoding_type):
```

Comment:
Shouldn't this function be called `set_encoding`?
```suggestion
    def set_encoding(self, encoding_type):
```